CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -211,9 +217,7 @@ local function isTenFootInterface()
 end
 
 local function usesSelectedObject()
-	if UserInputService.TouchEnabled and not UserInputService.GamepadEnabled then return false end
-
-	return true
+	return not (UserInputService.TouchEnabled and not UserInputService.GamepadEnabled or UserInputService.VREnabled)
```

Comment:
This statement is starting to get a little difficult to parse and you need to think about the order of operations with `or` and `and` statements. Do you think we can add some more parens to make it more obvious?